UI/Pages/TuitionPartner.cshtml

```diff
@@ -3,305 +3,392 @@
+      <div>
+        <h2 class="govuk-heading-m">Tuition cost information</h2>
 
-    <p class="govuk-body">
-        Each pupil premium eligible student will receive £162. This funding cost has been calculated so you will have 60% of the cost covered for up to £18 per pupil per hour. For help understanding your funding, you can see our information on <a href="/funding-and-reporting" class="govuk-link" data-testid="funding-guidance-1">funding and reporting</a>.
-    </p>
+        <p class="govuk-body">
+          Each pupil premium eligible student will receive £162. This funding cost has been calculated so you will have 60% of the cost covered for up to £18 per pupil per hour. For help understanding your funding, you can see our information on <a href="/funding-and-reporting" class="govuk-link" data-testid="funding-guidance-1">funding and reporting</a>.
+        </p>
 
-    <div>
-        <govuk-details data-testid="payment-details">
+        <div>
+          <govuk-details data-testid="payment-details">
             <govuk-details-summary>
-                How much will I pay?
+              How much will I pay?
             </govuk-details-summary>
             <govuk-details-text>
-                <strong>Example:</strong>
-                <ul class="govuk-list govuk-list--bullet">
-                    <li>If you spend £15 per pupil per hour, the government will cover 60% of the cost</li>
-                    <li>If you spend £25 per pupil per hour, you will receive 60% of £18 and will have to pay the rest</li>
-                </ul>
+              <strong>Example:</strong>
+              <ul class="govuk-list govuk-list--bullet">
+                <li>If you spend £15 per pupil per hour, the government will cover 60% of the cost</li>
+                <li>If you spend £25 per pupil per hour, you will receive 60% of £18 and will have to pay the rest</li>
+              </ul>
             </govuk-details-text>
-        </govuk-details>
-    </div>
-</div>
+          </govuk-details>
+        </div>
+      </div>
+
+      <h2 class="govuk-heading-m">Tuition prices</h2>
+      <p class="govuk-body">
+        Prices shown are per pupil per hour.<br />
+        <span show-if="!@Model.Data.HasPricingVariation" data-testid="pricing-same-for-subjects">
+          All subjects are the same price but depend on tuition group size.
+        </span>
+        <span show-if="@Model.Data.HasPricingVariation" data-testid="pricing-differences-for-subjects">
+          Some prices are different depending on key stage or subject. Please contact @Model.Data.Name for more information.
+        </span>
+        @if (!Model.Data.IsVatCharged)
+        {
+          <div class="govuk-inset-text">
+            <p class="govuk-body">This tuition partner does not charge VAT.</p>
+            <p class="govuk-body">Tuition partners such as charities or Limited Liability Partnerships (LLP) do not charge VAT.</p>
+          </div>
```

Comment:
We can add the logic for the one price table when a TP does not charge VAT here:

```
        <table class="govuk-table" data-testid="pricing-table">
          <caption class="govuk-visually-hidden">Tuition prices (excluding VAT)</caption>
          <thead class="govuk-table__head">
            <tr class="govuk-table__row">
              <th scope="col" class="govuk-table__header">Tuition group size</th>
              <th show-if="@Model.Data.Prices.ContainsInSchoolPrice()" scope="col" class="govuk-table__header">In school</th>
              <th show-if="@Model.Data.Prices.ContainsOnlinePrice()" scope="col" class="govuk-table__header">Online</th>
            </tr>
          </thead>
          <tbody class="govuk-table__body">
            @foreach (var item in Model.Data.Prices)
            {
              <tr class="govuk-table__row">
                <th scope="row" class="govuk-table__header" data-testid="pricing-group-size-column">
                  @(
                      ((Domain.Enums.GroupSize)item.Key).DisplayName()
                      )
                </th>
                <td show-if="@Model.Data.Prices.ContainsInSchoolPrice()" class="govuk-table__cell">
                  @item.Value.FormatFor(Domain.Enums.TuitionType.InSchool, false)
                </td>

                <td show-if="@Model.Data.Prices.ContainsOnlinePrice()" class="govuk-table__cell">
                  @item.Value.FormatFor(Domain.Enums.TuitionType.Online, false)
                </td>
              </tr>
            }
          </tbody>
        </table>
```

Reply:
I'm not sure that I understand this one?  The scenario where the TP does not charge VAT is already catered for, with the following code or was the comment meaning something else:

```
      @if (!Model.Data.AllPrices.Any())
      {
        <h3 class="govuk-heading-s" show-if="@Model.Data.IsVatCharged">Tuition prices (excluding VAT)</h3>
        <table class="govuk-table" data-testid="pricing-table">
...
```

Reply:
Great, I understand what you're saying. By default, we only display the title if a TP charges VAT for the pricing table. However, if a TP does charge VAT, we display both the including and excluding prices, along with the title for the table.